backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5752670_sys_BusinessRule_C_UOM_Conversion_CatchWeight.sql

```diff
@@ -74,3 +63,26 @@ INSERT INTO AD_BusinessRule_Precondition (AD_BusinessRule_ID,AD_BusinessRule_Pre
 INSERT INTO AD_BusinessRule_Trigger (AD_BusinessRule_ID,AD_BusinessRule_Trigger_ID,AD_Client_ID,AD_Org_ID,Created,CreatedBy,IsActive,OnDelete,OnNew,OnUpdate,Source_Table_ID,TargetRecordMappingSQL,Updated,UpdatedBy) VALUES (540014,540019,0,0,TO_TIMESTAMP('2025-04-24 08:29:44.398000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'Y','N','Y','Y',175,'C_UOM_Conversion_ID',TO_TIMESTAMP('2025-04-24 08:29:44.398000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100)
 ;
 
+-- 2025-04-25T09:25:50.334Z
+DELETE FROM AD_BusinessRule_Precondition WHERE AD_BusinessRule_Precondition_ID=540008
+;
+
+-- 2025-04-25T09:30:27.690Z
+DELETE FROM T_ES_FTS_Search_Result t 
```

Comment:
wtf is this?

Reply:
consider creating an mf15 issue to avoid logging T_ES_FTS_Search_Result table

Reply:
issue created